### PR TITLE
Fix formatting in issue 610

### PR DESCRIPTION
**Pull request by dpino**
Additional patch for fixing formatting in issue #610 (igalia/snabb).

### CODE REVIEW

File: src/program/snabbvmx/doc/README.troubleshooting.md [hunk 1]
```diff
@@ -320,10 +320,37 @@ NOTE: Currently the test is not working correctly: the returned MAC should be
...
+
+(5) run the scripted test:
+
+`snabb/src/program/snabbvmx/tests/end-to-end$ sudo ./end-to-end.sh`
+
+
+### How the system test works in detail
+
+The way the test system works is by passing the input IPv4/IPv6 packets (via pre-recorded pcap-files) to the lwAFTR and comparing the expected packet output (pcap-file) to the packet output that the lwAFTR has generated.
+The optional counters file is compared too to the actual counters file obtained after running the test. This is beeing reflected in snabbvmx check syntax:
```
Comment (teknico) [inline src/program/snabbvmx/doc/README.troubleshooting.md]: `beeing` -> `being`, or even remove it entirely.

File: src/program/snabbvmx/doc/README.troubleshooting.md [hunk 2]
```diff
@@ -320,10 +320,37 @@ NOTE: Currently the test is not working correctly: the returned MAC should be
...
+
+`CONF V4-IN.PCAP V6-IN.PCAP V4-OUT.PCAP V6-OUT.PCAP [COUNTERS.LUA]`
+
+V4-OUT.PCAP, V6-OUT.PCAP and COUNTERS.LUA are expected output. These output is compared to files stored temporarily in /tmp/endoutv4.pcap, /tmp/endoutv6.pcap and /tmp/counters.lua. These temporal files are the actual output produced by the lwAFTR after running a test. In order for a test to pass, the actual output must match the expected output. So it's not only that the counters file should match, but also the output .pcap files. (read: if a counters.lua file is provided, then it must still match the V4-OUT.PCAP and V6-OUT.PCAP)
```
Comment (teknico) [inline src/program/snabbvmx/doc/README.troubleshooting.md]: `expected output` -> `expected outputs`
`These output is compared` -> `These outputs are compared`
`These temporal files` -> `These temporary files`

File: src/program/snabbvmx/doc/README.troubleshooting.md [hunk 3]
```diff
@@ -370,38 +400,35 @@ Usage: check [-r] CONF V4-IN.PCAP V6-IN.PCAP V4-OUT.PCAP V6-OUT.PCAP
...
-- **V4-OUT.PCAP**: Outgoing IPv4 packets (to Internet, decapsulated).
-- **V6-OUT.PCAP**: Outgoing IPv6 packets (to b4, encapsulated)
-- **[COUNTERS.LUA]**: Lua file with counter values. Will be regenerated via [-r] parm
-
+- **CONF** : SnabbVMX (icmp_snabbvmx-lwaftr-xe.cfg) or lwaftr (icmp_snabbvmx-lwaftr-xe1.conf) configuration file.
+- **V4-IN.PCAP** : Incoming IPv4 packets (from Internet).
+- **V6-IN.PCAP** : Incoming IPv6 packets (from b4).
+- **V4-OUT.PCAP** : Outgoing IPv4 packets (to Internet, decapsulated).
+- **V6-OUT.PCAP** : Outgoing IPv6 packets (to b4, encapsulated)
+- **[COUNTERS.LUA]** : Lua file with counter values. Will be regenerated via [-r] param
```
Comment (teknico) [inline src/program/snabbvmx/doc/README.troubleshooting.md]: Why adding a blank before colons in the above six lines? It does not look good.

File: src/program/snabbvmx/doc/README.troubleshooting.md [hunk 4]
```diff
@@ -370,38 +400,35 @@ Usage: check [-r] CONF V4-IN.PCAP V6-IN.PCAP V4-OUT.PCAP V6-OUT.PCAP
...
 
 ## How to run SnabbVMX interactive end-to-end test
 
-If you detected an error in the lwAFTR, the first step is to obtain the
-configuration file that SnabbVMX was using, as well as a copy of lwAFTR's
-configuration and binding table.  With that information and knowing the error
-report (ping to lwAFTR but it doesn't reply, valid softwire packet doesn't get
-decapsulated, etc), you craft a hand-made packet that meets the testing case.
+If you detected an error in the lwAFTR, the first step is to obtain the configuration file that SnabbVMX was using, as well as a copy of lwAFTR's configuration and binding table.  With that information and knowing the error report (ping to lwAFTR but it doesn't reply, valid softwire packet doesn't get decapsulated, etc), you craft a hand-made packet that meets the testing case.
```
Comment (teknico) [inline src/program/snabbvmx/doc/README.troubleshooting.md]: I don't like very long lines: they make diffs harder to read. I prefer hard-wrapped lines.

File: src/program/snabbvmx/doc/README.troubleshooting.md [hunk 5]
```diff
@@ -441,24 +468,41 @@ lab@ubuntu1:~/vmxlwaftr/tests/t1$ tar -tvzf support-info-20161108-1335.tgz
...
+vFPC-20160922.img           xe0.socket
+vhost_features_xe0.socket   xe1.socket
 ```
-Note: press ctrl p ctrl q to exit the containers shell
 
+Note: Press Ctrl-P + Ctrl-Q to exit the containers shell.
+
+**Some adoption of config-files is required**
+
+The advantage of snabbvmx is the dynamic next-hop resolution via Junos. When running the the lwaftr or snabbvmx app standalone, then the next-hop resolution via Junos is missing. The config-files are required to get modified for static next-hop configuration.
```
Comment (teknico) [inline src/program/snabbvmx/doc/README.troubleshooting.md]: `When running the the lwaftr` -> `When running the lwaftr`

File: src/program/snabbvmx/doc/README.troubleshooting.md [hunk 6]
```diff
@@ -550,105 +596,91 @@ softwires {
...
+- Binding-table.
+- Input pcaps.
+- Expected resulting counters and out.pcap.
 
-At this stage the interactive test is finished.
-The following is defined:
-- lwaftr and  snabbvmx configs
-- binding-table
-- input pcaps
+With known input and results, the test can now be added to the scripted end-to-end.sh to be executed with all other tests to ensure snabbvmx behaves as it should. Furthermore, this end-to-end procedure can be ideally used to report issues!
```
Comment (teknico) [inline src/program/snabbvmx/doc/README.troubleshooting.md]: `to the scripted end-to-end.sh` -> `to the "end-to-end.sh" script`

**Review by teknico — CHANGES_REQUESTED** [review]
As mentioned inline, I don't like very long lines in documentation: they make diffs harder to read. I prefer hard-wrapped lines.

**Comment by dpino** [comment]
Thanks for the review @teknico . Fixed the issues and amended the commented issues. PTAL.

**Review by teknico — APPROVED** [review]
Thank you for wrapping those long lines.

**Review by chrgraf — APPROVED** [review]
@dpino @teknico 
Thanks. I will avoid the large lines in future.

**Pull request closed without merging**